bndtools.m2e/src/bndtools/m2e/BndConfigurator.java

```diff
@@ -1,5 +1,7 @@
 package bndtools.m2e;
 
+import static java.util.Optional.ofNullable;
```

Comment:
I generally don't like the static import of simple methods like `of`, `ofNullable` since you don't know what they really do from reading the code below. There is Map.of, MapStream.of, etc...